test/e2e/testdata/data-transformation.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
```

Comment:
I think you could make this a _testable example_ - BUT

a _testable example_ should make a good example, if you need to be more complex, then better to have both examples and test

Reply:
> if you need to be more complex, then better to have both examples and test

I think in this case we need to be more complex. See comment: https://github.com/argoproj/argo-workflows/pull/4958#discussion_r583057809